packages/mobile/src/store/offline-downloads/sagas/downloadQueueSagas/workers/downloadTrackWorker.ts

```diff
@@ -70,7 +77,8 @@ function* downloadTrackAsync(
 
   const track = yield* call([apiClient, apiClient.getTrack], {
     id: trackId,
-    currentUserId
+    currentUserId,
+    abortOnUnreachable: false
```

Comment:
So what actually happens when we go unreachable is we don't fail here, but the saga will be cancelled soon after?

Reply:
Maybe worth a comment here about why this flag is important

Reply:
yeah exactly, cause what was happening is that it would try and immediately return null, which results in us calling ERROR in the job. def will add a comment!